src/component/FeatureRenderer.js

```diff
@@ -62,36 +62,36 @@
  *
  * @class GeoExt.component.FeatureRenderer
  */
-Ext.define('GeoExt.component.FeatureRenderer', {
-    extend: 'Ext.Component',
-    alias: 'widget.gx_renderer',
+Ext.define("GeoExt.component.FeatureRenderer", {
```

Comment:
Using double quotes consistently instead of single quotes is a big change. But with ES2015 template strings single quotes really lend to be confused.